build/unix/makepchinput.py

```diff
@@ -230,37 +230,37 @@ def isDirForPCH(dirName):
    Check if the directory corresponds to a module whose headers must belong to
    the PCH
    """
-   PCHPatternsWhitelist = ("interpreter/",
-                           "core/",
-                           "io/io",
-                           "net/net",
-                           "math/",
-                           "hist/",
-                           "tree/",
-                           "graf2d",
-                           "graf3d/ftgl",
-                           "graf3d/g3d",
-                           "graf3d/gl",
-                           "gui/gui",
-                           "gui/fitpanel",
-                           "rootx",
-                           "bindings/pyroot",
-                           "roofit/",
-                           "tmva",
-                           "main")
-   PCHPatternsBlacklist = ("graf2d/qt",
-                           "gui/guihtml",
-                           "gui/guibuilder",
-                           "math/fftw",
-                           "math/foam",
-                           "math/fumili",
-                           "math/mlp",
-                           "math/quadp",
-                           "math/rtools"
-                           "math/splot",
-                           "math/unuran",
-                           "math/vdt",
-                           "tmva/rmva")
+   PCHPatternsWhitelist = (os.path.normpath("interpreter/"),
```

Comment:
Can you run `os.path.normpath` in `isAnyPatternInString()`? Seems a lot less verbose...

Reply:
Good point! I'll try and commit the changes